test/helpers/IntegrationHelper.js

```diff
@@ -754,6 +754,42 @@ function generatePromptHelpMessage(player) {
     return `Current prompt for ${player.name}:\n${formatPrompt(player.currentPrompt(), player.currentActionTargets)}`;
 }
 
+function validatePlayerOptions(playerOptions, playerName, startPhase) {
+    // list of approved property names
+    const listOfApprovedPropertiesForAnyOtherPhase = [
```

Comment:
Let's say "ForNonSetupPhase" so this is understandable without seeing the other variable name. Also, it's personal preference but I think "listOf" in the name is kind of uncessary, if the name is plural it implies a list imo

Reply:
ah yes thank you i was thinking how i can rename this sausage of a variable.